elastic/integrations
# [netflow] Fix dashboard navigation links

## Proposed commit message



In certain contexts, the navigation panel links on the netflow dashboard were not functioning correctly. This issue has been rectified by replacing the navigation panel with a 'links' panel, which is now functional in all contexts.

## Checklist


## How to test this PR locally



Install the integration package, various contexts open the Netflow dashboards, and use the navigation panel to switch between the dashboards.  All links should work.

The contexts to try are: Analytics->dashboards, Security->dashboards, Integrations->Installed Integrations->Netflow Records->Assets.

## Related issues


- Closes #11448
